search/includes/classes/class-health.php

```diff
@@ -836,8 +836,24 @@ public static function get_last_post_id() {
 		global $wpdb;
 
 		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
-		$last = $wpdb->get_var( "SELECT MAX( `ID` ) FROM $wpdb->posts" );
+		$last_db_id = $wpdb->get_var( "SELECT MAX( `ID` ) FROM $wpdb->posts" );
 
+		$indexable = \ElasticPress\Indexables::factory()->get( 'post' );
+
+		$es_result = false;
+		if ( $indexable ) {
+			$query_args     = [
+				'posts_per_page' => 1,
+				'orderby'        => 'ID',
```

Comment:
I think it's better to be explicit here.

```suggestion
				'orderby'        => 'ID',
				'order'        => 'desc',
```